www/generate_tutorial/src/input/tutorial.md

```diff
@@ -1982,4 +1982,4 @@ Here are various Roc expressions involving operators, and what they desugar to.
 
 These are all of the language keywords supported by Roc;
 
-`if`,`then`,`else`,`when`,`as`,`is`,`dbg`,`expect`,`expect-fx`,`crash`,`interface`,`app`,`package`,`platform`,`hosted`,`exposes`,`imports`,`with`,`generates`,`package`,`packages`,`requires`,`provides`,`to`
+`if`,`then`,`else`,`when`,`as`,`is`,`dbg`,`expect`,`expect-fx`,`crash`,`interface`,`app`,`package`,`platform`,`hosted`,`exposes`,`imports`,`with`,`generates`,`packages`,`requires`,`provides`,`to`
```

Comment:
So,`package` is used when declaring a package:
```
package "parser"
    exposes [
        ParserCore,
        ParserCSV,
        ParserStr,
        ParserHttp,
    ]
    packages {}
``` 
And `packages` is used when "importing" packages and  a platform:
```
app "example"
    packages {
        pf: "../../platform-switching/zig-platform/main.roc",
        parser: "../package/main.roc",
    }
    imports [
        parser.ParserCore.{ Parser, map, keep },
        parser.ParserStr.{ RawStr, strFromRaw },
        parser.ParserCSV.{ CSV, record, field, string, nat, parseStr },
    ]
    provides [main] to pf
```
`packages` does definitely not feel optimal, especially because the platform is not a package. I can't think of anything better right now though.

Reply:
Oh sure, I just meant that it isn't a duplicate keyword. It is two separate keywords, so I don't think it makes sense to submit this PR. I do agree that naming could be changed to make this better.

Reply:
Oh, nevermind. I missed the duplicate. it is `package` twice and `packages` onces.